config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  scope "(:locale)", locale: /en|vi/ do
+    root "static_pages#home"
+    get "/help", to:"static_pages#help"
```

Comment:
sau dấu `:` em để 1 dấu cách nhé, fix tương tự bên dưới
```suggestion
    get "/help", to: "static_pages#help"
```